docker/unlock-core.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:8.11.4-alpine
```

Comment:
This new image is our "parent" image with all the stuff that is shared between all other images.